src/components/Hero.js

```diff
@@ -2,24 +2,27 @@ import React, { useState } from "react";
 import styled from "styled-components";
 import { IoMdCloseCircle } from "react-icons/io";
 
-const close = (isVisible, setIsVisible) => {
-  setIsVisible(!isVisible);
-}
+import { setLocalData, getLocalData } from "../utils/localdata"
 
 export default function Hero(props) {
-  const [isVisible, setIsVisible] = useState(true)
+  const [isClosed, setIsClosed] = useState(getLocalData('verifactHeroClosed'))
   const welcomeTitle = "Your home for verifying credible news"
   const welcomeContent = "Post a question to our community of news sleuths to get answers and new perspectives about the news your reading"
 
-  return <>{isVisible ? (<Wrapper>
+  const close = () => {
+    setLocalData('verifactHeroClosed', 'true')
```

Comment:
Because we're serializing anything that's not a string and parsing those values, you can now pass the boolean `true` instead of the string `'true'` to make to make it clear what you expect when retrieving data.